lib/puppet/type/zpool.rb

```diff
@@ -85,7 +85,7 @@ def insync?(is)
 
     validate do
       has_should = [:disk, :mirror, :raidz].select { |prop| self.should(prop) }
-      self.fail "You cannot specify #{has_should.join(" and ")} on this type (only one)" if has_should.length > 1
+      self.fail _("You cannot specify #{has_should.join(" and ")} on this type (only one)") if has_should.length > 1
```

Comment:
This is kind of dicey - strictly speaking it adheres to the rules, but it results in this string to be translated:
```
msgid "\"You cannot specify #{has_should.join(\" and \")} on this type (only one)\""
```
As long as the translator's not confused by the double quotes inside the string it's ok.